x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ssl/SSLService.java

```diff
@@ -501,7 +501,8 @@ private SSLContextHolder createSslContext(
             sslContext.init(new X509ExtendedKeyManager[] { keyManager }, new X509ExtendedTrustManager[] { trustManager }, null);
 
             // check the supported ciphers and log them here to prevent spamming logs on every call
-            supportedCiphers(sslContext.getSupportedSSLParameters().getCipherSuites(), sslConfiguration.getCipherSuites(), true);
+            // FIXME error log for unsupported ciphers on JDK 24 breaks packaging tests (see ES-10984)
+            supportedCiphers(sslContext.getSupportedSSLParameters().getCipherSuites(), sslConfiguration.getCipherSuites(), false);
```

Comment:
@rjernst this is the error logger that breaks the packaging tests. I've tried fixing those tests to be more robust. Imho they shouldn't care about the logger, but didn't get far.